wrangler-js/index.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+const webpack = require("webpack");
+const { join } = require("path");
+const { writeFileSync } = require("fs");
+
+const rawArgs = process.argv.slice(2);
+const args = rawArgs.reduce((obj, e) => {
+  if (e.indexOf("--") === -1 && e.indexOf("=") === -1) {
+    throw new Error("malformed arguments");
+  }
+
+  const [name, value] = e.split("=");
+  const normalizedName = name.replace("--", '');
+  obj[normalizedName] = value;
+  return obj;
+}, {});
+
+let config;
+if (args["no-webpack-config"] === "1") {
+  config = { entry: args["use-entry"] };
+} else {
+  config = require(join(process.cwd(), "./webpack.config.js"));
+}
+
+if (config.target !== "webworker") {
+  if (config.target !== undefined) {
+    console.warn("Unsupported Webpack target; webworker has been set.");
+  }
+  config.target = "webworker";
+}
+const compiler = webpack(config);
```

Comment:
Will relying on an in-memory [filesystem for for webpack ](https://webpack.js.org/api/node/#custom-file-systems) allow us not having a `dist_to_clean`?

Reply:
I made #116, that's a good question.